docs/import/list_ee_assets.js

```diff
@@ -35,7 +36,36 @@ function getDisasterAssetsFromEe(disaster) {
   if (maybePromise) {
     return maybePromise;
   }
-  const result = listEeAssets(eeLegacyPathPrefix + disaster);
+  const result = listEeAssets(eeLegacyPathPrefix + disaster)
+                     .then((result) => {
+                       const assetPromises = [];
+                       result.forEach((type, asset) => {
+                         if (type !== LayerType.FEATURE_COLLECTION) return;
+                         // assuming that if the first feature's geometry in a
+                         // collection is null, they're all null
+                         const assetPromise = convertEeObjectToPromise(
+                                             ee.FeatureCollection(asset)
+                                                 .first()
+                                                 .geometry()
+                                                 .coordinates())
+                                             .then((coords) => {
+                                               if (coords.length !== 0) {
+                                                 return [asset, type];
```

Comment:
nit: return null here instead of the implicit undefined? Prospective nit: in that case, just do a ternary, or do the positive `if (coords.length === 0) { return null; }`